docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/cards/ncids_card_macros.twig

```diff
@@ -26,3 +26,26 @@
 		</picture>
 	{% endif %}
 {% endmacro %}
+
+{% macro ncidsFlagCard(title, description, image, url, directory, id, entity_type, link_type) %}
+	{% set renderImage = image|render %}
+	{% import _self as self %}
+  {% set titleID = "#{entity_type}-#{id}" %}
+	{# pass in node id? *check xdebug* aria-labelledby#}
```

Comment:
This should be removed, right?

Reply:
@dev-rana-publicis please make the change as Bryan wrote above. The requirement is, 

> "The Card Title font shall be the same size visually and in the page hierarchy as the title on updated Feature Cards (as of writing, not all Feature Cards have been updated to the new styling - see Specifications)"

Specifications:

> - The Flag Card Group heading will be an h2 element
> - The Flag Card Group heading will stylistically use token size xl (24px)
> - The Flag Card Title will stylistically use token size md (20px)
> - The Flag Card Description will stylistically use token size xs (16px)